src/Resources/views/easy_admin/field_url.html.twig

```diff
@@ -0,0 +1,6 @@
+{% if view == 'show' %}
```

Comment:
This modification will replace all url templates usage. Try to investigate the possibility to replace the template only for one field. Try to rename this template to torunament_field_ur.html.twig and configure the field like this:
fields: ['id', {property: 'link', type: 'url',  template: 'torunament_field_ur.html.twig'}, ....